packages/amplify-provider-awscloudformation/src/amplify-service-manager.js

```diff
@@ -133,15 +134,12 @@ async function init(amplifyServiceParams) {
     } catch (e) {
       log(e);
       if (e.code === 'LimitExceededException') {
-        // Do nothing
-      } else if (
-        e.code === 'BadRequestException' &&
-        e.message.includes('Rate exceeded while calling CreateApp, please slow down or try again later.')
-      ) {
-        // Do nothing
+        printer.error('You have reached the Amplify App limit for this account and region')
```

Comment:
Will this affect our e2e test runs since it wont allow a undefined appId ?

Reply:
Just kicked off an e2e run so we'll see. But it shouldn't. Most of our e2e tests don't even create an app id and the ones that do will fail for some other reason if one can't be created